compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -1428,6 +1430,9 @@ object Types {
     /** If this is an ignored proto type, its underlying type, otherwise the type itself */
     def revealIgnored: Type = this
 
+    /** If this is a proto type, the ignored version, otherwise the type itself */
```

Comment:
I would expect "ignored version" to mean "IgnoredProto" but we always map to WildcardType instead, so maybe the documentation should read:
```suggestion
    /** If this is a proto type, WildcardType, otherwise the type itself */
```